packages/block-editor/src/components/block-draggable/index.native.js

```diff
@@ -181,15 +190,21 @@ const BlockDraggableWrapper = ( { children } ) => {
 		chip.x.value = dragPosition.x;
 		chip.y.value = dragPosition.y;
 
+		hasStartedDraggingOver.value = true;
```

Comment:
I'm wondering if we could use the shared value `isDragging` to know if the dragging has started instead of defining a new one (i.e. `hasStartedDraggingOver`). I think there won't be a big difference because handler `startDragging` will be called once the dragging gesture starts, wdyt?

Reply:
I updated the code in ef85bf909f1d832aea53cb5c4c60d0a82f5a27f6